app/workers/reminder_worker.rb

```diff
@@ -0,0 +1,26 @@
+# TODO: Move worker logic into service
+require 'post_to_slack'
+class ReminderWorker
+  include Sidekiq::Worker
+  def perform
+
+    # First check there is an active question
```

Comment:
We should use guard clauses instead of nesting conditionals. If there are no open questions or no active users we can `return` from this method. Then the following code will be much more straightforward to read